### DISCUSSION TITLE
Create a starter script/cli

### DISCUSSION THREAD

**Discussion by rexhent** [TanStack/router#2414] [Ideas]
Such as npx create-tanstack-start or npm create @tanstack/start

Similar to the way next, remix, nuxt do it.

Nuxt has a cli which has the init command included. It is built of https://github.com/unjs/citty which could be a good solution for tanstack start and router?

**Comment by SeanCassiere** [comment]
We've got a starter command for Router (as shown in the docs) and it's on the list of things todo for Start.
